### ISSUE TITLE
Diagram option 'no-doctype': strange behaviour

### ISSUE THREAD

**Issue by deining** [yuzutech/kroki#1465]
As documented [here](https://docs.kroki.io/kroki/setup/diagram-options/#_blockdiag), you can suppress the doctype definition tag for your `blockdiag` diagram (`svg`) using the `no-doctype` diagram option. This works for me only partly, however:

```
$ http https://kroki.io/ diagram_type='actdiag' output_format='svg' diagram_source='{lane Sysadmin}' diagram_options:='{"no-doctype": true  }'
```

This correctly outputs an `svg` image **without** doctype definition tag.

Now I set option `no-doctype` to `false`:

```
$ http https://kroki.io/ diagram_type='actdiag' output_format='svg' diagram_source='{lane Sysadmin}' diagram_options:='{"no-doctype": false  }'
```
**Expected behaviour**: with `no-doctype` set to `false`, doc-type should be shown.
**Actual behaviour**: doc-type is **not** shown.

Is this the intended behaviour?

**Comment by ggrossetie** [comment]
I would say yes.
If you want the doctype then do not pass the option `no-doctype`. 

It's shorter to type:

```console
$ http https://kroki.io/ diagram_type='actdiag' output_format='svg' diagram_source='{lane Sysadmin}'
```

than:

```console
$ http https://kroki.io/ diagram_type='actdiag' output_format='svg' diagram_source='{lane Sysadmin}' diagram_options:='{"no-doctype": false }'
```

And the double negation makes it difficult to read. 
Also boolean parameter in the HTTP URL are often just a flag: `?no-doctype` (and not `?no-doctype=true`)

**Comment by ggrossetie** [comment]
I don't remember how we handle `no-transparency` since the default value is `true`. Does `no-transparency=false` works? If so, we might support boolean values for consistency.

**Comment by deining** [comment]
> Does `no-transparency=false` works?

`no-transparency=false` and `no-transparency=true` both behave the same way: they bring the option `no-transparency` into effect.

**Comment by deining** [comment]
>>Is this the intended behaviour?

> I would say yes. If you want the doctype then do not pass the option `no-doctype`.

I agree. This is a minor issue that does not justify a code change. I just submitted PR #1469 that clarifies things in the documentation. I think that's all we need to do here.
